app/views/library/LibraryMainView.vue

```diff
@@ -0,0 +1,322 @@
+            </div>
+          </div>
+        </div>
+        <div class="row library__needs__info library__needs-only-text">
+          <div class="col-md-11 library__needs__info__text">
+            <div class="library__needs__info__text-heading">
+              Usage Stats
+            </div>
+            <div class="library__needs__info__text-sub">
+              Gain insight to your members’ usage with our snapshot reporting tool. It provides access to usage stats at any time, including monthly active users, licensed days used and more valuable data points.
+            </div>
+          </div>
+        </div>
+        <div class="row library__needs__info library__needs-only-text">
+          <div class="col-md-11 library__needs__info__text">
+            <div class="library__needs__info__text-heading">
+              Academic Esports
+            </div>
+            <div class="library__needs__info__text-sub">
+              Our esports program offers an opportunity to further engage your members through a fun and exciting competition. The AI League allows members to showcase their understanding of coding concepts in a new environment and practice problem solving strategies as they go head to head in battle against the global community.
+            </div>
+          </div>
+        </div>
+        <div class="library__continue">
+          <img class="img-responsive" src="/images/pages/parents/graphic_07.svg" style="margin: 0 auto; transform: translate(-47%, 0);" loading="lazy"/>
+        </div>
+        <section class="library__footer">
+          <div class="library__footer-content row">
+            <div class="col-md-offset-2 col-md-8">
+              <h2 class="library__footer-text pixel-text">
+                Work with us to unlock the next generation of engineers, programmers and CS professionals in your community.
+              </h2>
+              <div class="library__footer-cta">
+                <button class="btn library__footer-cta-btn btn-moon" @click="toggleLicenseModal">
+                  Schedule a Demo
+                </button>
+              </div>
+            </div>
+          </div>
+        </section>
+      </section>
+    </div>
+    </div>
+    <modal-get-licenses
+      v-if="showLicenseModal"
+      subtitle="Please complete this short form and one of our library specialists will reach out to provide a demo and custom pricing options!"
+      email-message="Hi CodeCombat team! I’m interested in learning more about CodeCombat and how it can be used for my library or library network membership.
+
+        Name of Library or Library Network:
+        State or Region Served:
+        My Role:
+        My Phone Number:
+      "
+      :ask-school-info="askSchoolInfo"
+      licenses-needed-text="Members Needing Access"
+      licenses-needed-placeholder="Approximately how many members do you plan to provide access?"
```

Comment:
Grammar's off, would need another "to"; or just shorten:
```suggestion
      licenses-needed-placeholder="Approximately how many members will need access?"
```